src/view/moveable/moveable-types.js

```diff
@@ -12,7 +12,7 @@ export type Props = {|
   speed: Speed,
   destination: Position,
   onMoveEnd: () => void,
-  children: (Style) => Node,
+  children: (Style) => Element<*>,
```

Comment:
can you explain the reason for this change? We support any React node..

Reply:
In this case we inherit react motion types. As a workaround we may add fragment or just fix motion typed. Also `any` may fix this too)

Reply:
I'll defer to you for this one